css/src/tokens/colors.scss

```diff
@@ -7,6 +7,8 @@ $black-static: #000;
 $text: var(--theme-text);
 $text-subtle: var(--theme-text-subtle);
 $text-invert: var(--theme-text-invert);
+$text-purple: var(--theme-text-purple);
```

Comment:
We shouldn't be using color specific names, each thing should be a concept

Reply:
I got really confused about what to do with these, let's brainstorm offline!

Reply:
Removed!